django-stubs/contrib/admin/widgets.pyi

```diff
@@ -42,15 +42,26 @@ class ForeignKeyRawIdWidget(forms.TextInput):
     admin_site: AdminSite
     db: str | None
     def __init__(
-        self, rel: ManyToOneRel, admin_site: AdminSite, attrs: _OptAttrs | None = ..., using: str | None = ...
+        self,
+        rel: ManyToOneRel,
+        admin_site: AdminSite,
+        attrs: _OptAttrs | None = ...,
+        using: str | None = ...,
     ) -> None: ...
     def base_url_parameters(self) -> dict[str, str]: ...
     def get_context(self, name: str, value: Any, attrs: _OptAttrs | None) -> dict[str, Any]: ...
     def url_parameters(self) -> dict[str, str]: ...
     def label_and_url_for_value(self, value: Any) -> tuple[str, str]: ...
 
 class ManyToManyRawIdWidget(ForeignKeyRawIdWidget):
-    rel: ManyToManyRel  # type: ignore
+    rel: ManyToManyRel
```

Comment:
Did you notice that tests are failing?

You need to restore this `type: ignore`

Reply:
Actually I was in class at that time, thought I would look at the ci failures after reaching home 😅
Thanks for pointing this out.